beacon-chain/p2p/subnets.go

```diff
@@ -61,69 +135,88 @@ const dataColumnSubnetVal = 150
 // On some edge cases, this method may hang indefinitely while peers
 // are actually found. In such a case, the user should cancel the context
 // and re-run the method again.
-func (s *Service) FindPeersWithSubnet(ctx context.Context, topic string,
-	index uint64, threshold int) (bool, error) {
+func (s *Service) FindPeersWithSubnet(
+	ctx context.Context,
+	topic string,
+	index uint64,
+	threshold int,
+) (bool, error) {
+	const batchSize = 200
```

Comment:
could you make the batch bigger ? Ex: 1000 or 2000

Reply:
Fixed in 0b39d26be61abd7fc9dd00d780f03f2db4ba4e0e.